examples/bri-3/src/bri/communication/api/messages.controller.spec.ts

```diff
@@ -16,20 +16,24 @@ import { CreateBpiMessageDto } from './dtos/request/createBpiMessage.dto';
 import { UpdateBpiMessageDto } from './dtos/request/updateBpiMessage.dto';
 import { NOT_FOUND_ERR_MESSAGE } from './err.messages';
 import { NOT_FOUND_ERR_MESSAGE as BPI_SUBJECT_NOT_FOUND_ERR_MESSAGE } from '../../identity/bpiSubjects/api/err.messages';
-import Mapper from '../../utils/mapper';
+import { Mapper } from '@automapper/core';
 import { MessageController } from './messages.controller';
+import { AutomapperModule } from '@automapper/nestjs';
+import { classes } from '@automapper/classes';
+import { CommunicationProfile } from '../communicaton.profile';
+import { SubjectsProfile } from '../../identity/bpiSubjects/subjects.profile';
 
 describe('MessageController', () => {
   let mController: MessageController;
-
   let mockBpiMessageStorageAgent: MockBpiMessageStorageAgent;
   let mockBpiSubjectStorageAgent: MockBpiSubjectStorageAgent;
   let existingBpiSubject1: BpiSubject;
   let existingBpiSubject2: BpiSubject;
+  let mapper: Mapper;
 
   beforeEach(async () => {
-    mockBpiMessageStorageAgent = new MockBpiMessageStorageAgent(new Mapper());
-    mockBpiSubjectStorageAgent = new MockBpiSubjectStorageAgent(new Mapper());
+    mockBpiMessageStorageAgent = new MockBpiMessageStorageAgent(mapper);
```

Comment:
`mapper` is not defined here? `let mapper: Mapper` is declared, but how it is initialized?
EDIT: after checking the mock, mapper is never used there and thats why this undefined error is never thrown, you can remove mapper from mock and from here if its not used